python/caffe/draw.py

```diff
@@ -10,7 +10,16 @@
 """
 
 from caffe.proto import caffe_pb2
-import pydot
+
+"""
+pydot is not supported under python 3 and pydot2 doesn't work properly.
+pydotplus works nicely (pip install pydotplus)
+"""
+try:
+    # Try to load pydotplus
+    import pydotplus as pydot
+except:
```

Comment:
This should be `except ImportError` so that other types of exceptions are still raised.  Otherwise this LGTM.